cluster/gpu-provision-daemonset.yaml

```diff
@@ -1,48 +1,82 @@
+# Copyright 2017 Google Inc. All rights reserved.
```

Comment:
Do we need these for YAML?

Reply:
Dunno. I just copied this whole hog via 

`kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/k8s-1.9/nvidia-driver-installer/cos/daemonset-preloaded.yaml`